assets/cross-sell-carousel.js

```diff
@@ -0,0 +1,33 @@
+class CrossSellCarousel extends HTMLElement {
+  constructor() {
+    super();
+    this.glideInstance = new Glide('[data-glide]', {
+      type: 'carousel',
+      startAt: 0,
+      perView: 2,
+      gap: 15,
+      breakpoints: {
+        749: {
+          perView: 3,
+          gap: 20
+        },
+        480: {
+          perView: 2
+        },
+        305: {
+          perView: 1
+        }
+      }
+    })
+    this.mountGlideInstance();
```

Comment:
Using semi colons will depend on the team, in generall, there are very few instances where you would really need to use a semi colon.
I prefer not to use semi colons